### PR TITLE
Remove email campaign

### PR DESCRIPTION
**Pull request by surminus**
The application is no longer used.

### CODE REVIEW

File: modules/govuk/manifests/node/s_frontend_lb.pp [hunk 1]
```diff
@@ -22,7 +20,6 @@
     [
       'draft-collections',
       'draft-contacts-frontend',
-      'draft-email-alert-frontend',
```
Comment (alexmuller) [inline modules/govuk/manifests/node/s_frontend_lb.pp]: Not this one!

**Comment by surminus** [comment]
Thanks! Added draft-email-alert-frontend back in (thank god for peer review).

**Comment by alexmuller** [comment]
Looks good to me! There are also these files which can go:

- `./hieradata/node/email-campaign-mongo-2.api.integration.publishing.service.gov.uk.yaml`
- `./hieradata/node/email-campaign-mongo-2.api.publishing.service.gov.uk.yaml`
- `./hieradata/node/email-campaign-mongo-2.api.staging.publishing.service.gov.uk.yaml`

**Comment by alexmuller** [comment]
I've removed those additional files, can somebody else have another look over?

**Pull request merged**